src/Moryx.CommandCenter.Web/src/databases/container/DatabaseModel.tsx

```diff
@@ -198,7 +199,7 @@ class DatabaseModel extends React.Component<DatabaseModelPropsModel & DatabaseMo
             .then((response) => {
                 this.setState({
                     testConnectionPending: false,
-                    testConnectionResult: response.result !== undefined ? response.result : TestConnectionResult.ConnectionError
+                    testConnectionResult: response.result !== undefined ? getEnumValueForKey(TestConnectionResult, response.result) : TestConnectionResult.ConnectionError
```

Comment:
I think this was not the only place where this can cause an issue. Please verify all places the enum is used and checked against api calls 🙏